fcrepo-jms/src/main/java/org/fcrepo/jms/headers/DefaultMessageFactory.java

```diff
@@ -79,36 +93,23 @@ private void setBaseURL(final FedoraEvent event) {
                     url = url.substring(0, url.length() - 1);
                 }
                 this.baseURL = url;
-                LOGGER.debug("MessageFactory baseURL: {}", baseURL);
+                this.userAgent = json.get("userAgent").getAsString();
+                LOGGER.debug("MessageFactory baseURL: {}, userAgent: {}", baseURL, userAgent);
 
             } else {
-                LOGGER.warn("MessageFactory baseURL is empty!");
+                LOGGER.warn("MessageFactory event UserData is empty!");
             }
 
         } catch ( final Exception ex ) {
```

Comment:
I suppose that's a bit better. At least it won't trigger one of those
annoying Sonar warnings.

On Wed, Apr 1, 2015 at 11:44 AM, Esmé Cowles <notifications@github.com>
wrote:

> In
> fcrepo-jms/src/main/java/org/fcrepo/jms/headers/DefaultMessageFactory.java
> <https://github.com/fcrepo4/fcrepo4/pull/760#discussion_r27581705>:
>
> >
> >              } else {
> > -                LOGGER.warn("MessageFactory baseURL is empty!");
> > +                LOGGER.warn("MessageFactory event UserData is empty!");
> >              }
> >
> >          } catch ( final Exception ex ) {
>
> JsonParser.parse throws JsonParseException, but the accessors throw
> IllegalStateExceptions -- so we could narrow to RuntimeException.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/fcrepo4/fcrepo4/pull/760/files#r27581705>.
>